OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/operations/impl/executors/UpdateUserOperationExecutor.kt

```diff
@@ -131,6 +135,11 @@ internal class UpdateUserOperationExecutor(
                     deltasObject,
                 )
 
+                // deltas includes property that may impact IAM fetch
+                if (deltasObject.hasAtLeastOnePropertySet) {
```

Comment:
We want to use the new offset for any update, not just fields that support deltas. Example tags are not under delts but we want to wait for those.